cfme/tests/containers/test_containers_main_pages_sort.py

```diff
@@ -44,4 +43,13 @@ def test_containers_main_pages_sort(cls):
         sort_tbl.sort_by(header_text, 'descending')
         rows_descending = [r[col].text for r in sort_tbl.rows()]
 
-        assert rows_ascending[::-1] == rows_descending
+        expected_ascending_order = sorted(rows_ascending)
+        expected_descending_order = expected_ascending_order[::-1]
+
+        if expected_ascending_order == expected_descending_order:
```

Comment:
sorry for the misleading comment, i formed the it from just the surrounding code, missing the case of a number of equal elements